Dockerfile

```diff
@@ -42,6 +42,16 @@ WORKDIR /build/fairscale
 RUN git checkout fixing_memory_issues_with_keeping_overlap_may24
 RUN pip3 install -e .
 
+RUN pip install \
+        aim==3.16.2 \
```

Comment:
I think we can remove `aim` since it's more for monitoring of jobs than evaluation :)

Reply:
Ok, yea they also install this in setup.py `aim>=3.9.4` so I think it's redundant and would distract from PR conents